### DISCUSSION TITLE
Custom absolute path import not working

### DISCUSSION THREAD

**Discussion by ghost** [remix-run/remix#1714] [Q&A] [answered]
Hello,

I create a new remix app, and try to create a custom path for my component in tsconfig file like this:
```json
"baseUrl": ".",
    "paths": {
      "~/*": ["./app/*"],
      "@components/*": ["./app/components/*"]
},
```
so I can import my component like this:

```javascript
import MyComponent from "@components/MyComponent"
```

but it says cannot find module when build.

And interesting thing that i found is, if i change `~` to `@` in tsconfig file
```json
"paths": {
      "~/*": ["./app/*"],
}
```
to 

```json
"paths": {
      "@/*": ["./app/*"],
}
```
and import my component like this:
```javascript
import MyComponent from "@/components/MyComponent"
```

it says cannot find the module too.

I don't know if this is a remix issue or not,
can you guys help me, please?

Thank you

**Comment by sergiodxa** [comment]
Remix right now only supports `~/` imports, it doesn’t support any other short path configured in tsconfig or jsconfig files.

**Comment by machour** [comment, marked as answer]
This feature was added in #2412 and should be available in the next release 🎉

**Reply by ghost** [reply]
Thanks @machour 🎉 🚀
